randovania/data/json_data/prime3.txt

```diff
@@ -902,25 +935,30 @@ Asset id: 17479488334200413695
 > Dock to Port Observation Deck; Heals? False
   * Normal Door to Port Observation Deck/Dock to Auxiliary Lift
   > Dock to Security Access
-      Trivial
+      After Auxiliary Lift Elevator
   > Pickup (Missile Expansion)
       Trivial
 
 > Dock to Security Access; Heals? False
   * Normal Door to Security Access/Dock to Auxiliary Lift
   > Dock to Port Observation Deck
-      Grapple Lasso and Energy Cells Total
+      After Auxiliary Lift Elevator
+  > Event - Elevator
+      Grapple Lasso and Energy Cells Total ≥ 2
 
 > Pickup (Missile Expansion); Heals? False
   * Pickup 16; Major Location? False
   > Dock to Port Observation Deck
       Trivial
 
+> Event - Elevator; Heals? False
+  * Event Stairwell Ramps
+
```

Comment:
I think you picked the wrong event. This should be the elevator event, not stairwell. Also missing connection to bottom door.